.github/workflows/release.yaml

```diff
@@ -2,14 +2,45 @@ name: Release Packages and Docker images
 on:
   push:
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+'   # final release
+      - '[0-9]+.[0-9]+.[0-9]+-*' # prerelease/RC
 
 # Checks if any concurrent jobs is running for release CI and eventually cancel it.
 concurrency:
   group: ci-release
   cancel-in-progress: true  
 
 jobs:
+  release-settings:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get latest release
+        uses: rez0n/actions-github-release@v2.0
```

Comment:
I have one concern: our release process states that we push a tag and then open the draft release.
I think this step won't find the release when it gets triggered on tag push; eg:
* `git tag 0.35.0 && git push origin 0.35.0`
* the release workflow is started
* we haven't still opened the new release (and we would open it in draft btw: https://github.com/falcosecurity/falco/blob/master/RELEASE.md#2-update-the-github-release)
* the action will find `0.34.1` as latest release
* we are doomed

Reply:
I forgot about the possibility of a draft release! That's a great catch, thank you!

Reply:
Wait a second though, do we want to start this on manual tag or release creation? We could switch to start on GH release creation so that whether a tag already existed or not the process will be the exact same.

Reply:
The current release workflow starts on tags pushing, that what CircleCI does and what our release document states!

Reply:
Hey @LucaGuerra 
I recommend updating https://github.com/falcosecurity/falco/blob/master/RELEASE.md in this PR directly.

Reply:
Done!